.travis.yml

```diff
@@ -6,10 +6,19 @@ matrix:
       script: ./sbt ++$TRAVIS_SCALA_VERSION clean test
 
     - scala: 2.11.8
-      script: ./sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport mimaReportBinaryIssues
+      script: ./sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport mimaReportBinaryIssues docs/makeMicrosite
```

Comment:
this is required to check that all documentation still builds; `tut` actually compiles all the docs(!!), so regressions to the docs will cause a build failure here.